index.html

```diff
@@ -8,6 +8,7 @@
 
 
 	<h1>MIKE ELROD</h1>
+	<img src="/img/portfolio_mograph" alt="Portfolio Mograph" style="width: 200px; height: 200px;">

```

Comment:
should be portfolio_mograph.gif. Without the file extension, the browser will not know what file to look for. 

For example, you could have mograph.gif and mograph.jpg, so specificity is key.